### ISSUE TITLE
Can't install pact_verifier_cli

### ISSUE THREAD

**Issue by LukevanTricht** [pact-foundation/pact-reference#189]
I'm attempting to `cargo install pact_verifier_cli` on MacOS Big Sur, and the compilation is failing due to the below error:

```
error[E0658]: destructuring assignments are unstable
   --> /Users/lvantricht/.asdf/installs/rust/1.58.1/registry/src/github.com-1ecc6299db9ec823/pact_models-0.3.2/src/generators/time_expression_parser.rs:110:24
    |
110 |       (time_base, lex) = base(lex, exp, token)?;
    |       ---------------- ^
    |       |
    |       cannot assign to this expression
    |
    = note: see issue #71126 <https://github.com/rust-lang/rust/issues/71126> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `pact_models` due to previous error
```
The error appears to be changes that were made in pact_models 0.3.0.
I'm using Rust 1.58.1

**Comment by uglyog** [comment]
Can you upgrade Rust to 1.59.0?

**Comment by LukevanTricht** [comment]
I tried both `1.59.0` and `1.60.0`, but they both give the below error:
```
error[E0308]: mismatched types
   --> /home/lvantricht/.asdf/installs/rust/1.60.0/registry/src/github.com-1ecc6299db9ec823/pact_verifier_cli-0.9.9/src/main.rs:437:40
    |
437 |           PactSource::URL(v.to_string(), matches.value_of("user").map(|user| {
    |  ________________________________________^
438 | |           HttpAuth::User(user.to_string(), matches.value_of("password").map(|p| p.to_string()))
439 | |         }))
    | |__________^ expected enum `pact_models::http_utils::HttpAuth`, found enum `HttpAuth`
    |
    = note: expected enum `Option<pact_models::http_utils::HttpAuth>`
               found enum `Option<HttpAuth>`
    = note: perhaps two different versions of crate `pact_models` are being used?

error[E0308]: mismatched types
   --> /home/lvantricht/.asdf/installs/rust/1.60.0/registry/src/github.com-1ecc6299db9ec823/pact_verifier_cli-0.9.9/src/main.rs:441:40
    |
441 |         PactSource::URL(v.to_string(), matches.value_of("token").map(|token| HttpAuth::Token(token.to_string())))
    |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `pact_models::http_utils::HttpAuth`, found enum `HttpAuth`
    |
    = note: expected enum `Option<pact_models::http_utils::HttpAuth>`
               found enum `Option<HttpAuth>`
    = note: perhaps two different versions of crate `pact_models` are being used?

error[E0308]: mismatched types
   --> /home/lvantricht/.asdf/installs/rust/1.60.0/registry/src/github.com-1ecc6299db9ec823/pact_verifier_cli-0.9.9/src/main.rs:479:9
    |
479 |         auth,
    |         ^^^^ expected enum `pact_models::http_utils::HttpAuth`, found enum `HttpAuth`
    |
    = note: expected enum `Option<pact_models::http_utils::HttpAuth>`
               found enum `Option<HttpAuth>`
    = note: perhaps two different versions of crate `pact_models` are being used?

error[E0308]: mismatched types
   --> /home/lvantricht/.asdf/installs/rust/1.60.0/registry/src/github.com-1ecc6299db9ec823/pact_verifier_cli-0.9.9/src/main.rs:483:59
    |
483 |       PactSource::BrokerUrl(name, broker_url.to_string(), auth, vec![])
    |                                                           ^^^^ expected enum `pact_models::http_utils::HttpAuth`, found enum `HttpAuth`
    |
    = note: expected enum `Option<pact_models::http_utils::HttpAuth>`
               found enum `Option<HttpAuth>`
    = note: perhaps two different versions of crate `pact_models` are being used?

For more information about this error, try `rustc --explain E0308`.
error: failed to compile `pact_verifier_cli v0.9.9`, intermediate artifacts can be found at `/tmp/cargo-installwAP4gu`

Caused by:
  could not compile `pact_verifier_cli` due to 4 previous errors
```

**Comment by uglyog** [comment]
I've just released 0.9.10, can you try now with that version?

**Comment by LukevanTricht** [comment]
That worked, thank you!
